roles/filetree_create/templates/current_workflow_job_templates.j2

```diff
@@ -17,9 +17,17 @@ controller_workflows:
                      return_all=true, max_objects=query_controller_api_max_objects) %}
       - identifier: "{{ node.identifier }}"
         workflow_job_template: "{{ node.summary_fields.workflow_job_template.name }}"
+{% if 'approval' not in node.summary_fields.unified_job_template.unified_job_type %}
         unified_job_template: "{{ node.summary_fields.unified_job_template.name | default('ToDo: The node is pointing to a deleted Job Template') }}"
+{% endif %}
 {% if node.job_type %}
         job_type: "{{ node.job_type }}"
+{% if 'approval' in node.summary_fields.unified_job_template.unified_job_type %}
```

Comment:
I would prefer `node.summary_fields.unified_job_template.unified_job_type is match('approval')`